spec/feature/editing_card.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+feature "Editing cards" do
+  scenario "Try to edit a card" do
+    card = create(:card)
+    visit root_path
+
+    click_link "Карточки"
+    click_link "Редактировать", :match => :first
+    parsed_orig_text  = find("#card_original_text").value
```

Comment:
Useless assignment to variable - `parsed_orig_text`.<br>Unnecessary spacing detected.